gemdaq-testing/gemsupervisor/include/gem/supervisor/tbutils/ADCScan.h

```diff
@@ -195,8 +193,7 @@ namespace gem {
           uint32_t curDACValue;
           uint64_t stepSize_, samplesTaken_;
           bool is_working_, is_initialized_, is_configured_, is_running_;
-          vfat_shared_ptr vfatDevice_;
```

Comment:
Don't know why this got removed from yours, double check that it is in your release-v1 branch